src/taipy/core/data/_data_model.py

```diff
@@ -30,6 +30,7 @@ class _DataNodeModel:
     parent_ids: List[str]
     last_edit_date: Optional[str]
     job_ids: List[JobId]
```

Comment:
Should we just remove job_ids here?

Reply:
no, we should keep it so it can still read "legacy" data nodes. This is used later in the code. to migrate to the "edits".